featurebyte/api/target.py

```diff
@@ -177,6 +180,39 @@ def preview(  # pylint: disable=missing-function-docstring
     ) -> pd.DataFrame:
         return self._preview(observation_set=observation_set, url="/target/preview")
 
+    @typechecked
+    def info(self, verbose: bool = False) -> Dict[str, Any]:
+        """
+        Returns a dictionary that summarizes the essential information of an Target object. The dictionary
```

Comment:
didn't use the new annotation here yet, but will consider refactoring all the other `info` routes together.